isis/tests/data/jitterfit/S046mos1400x2250.def

```diff
@@ -0,0 +1,163 @@
+ Object = AutoRegistration
+   # Algorithm specification.
+   Group = Algorithm
+
+		 # This is the maximum correlation algorithm, which 
```

Comment:
If you want you can remove the comments. They came from the original template and have no direct correlation with this co-registration